models/PackageRequest.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose')
+const packageRequestSchema = new mongoose.Schema({
+    venue: { type: String, required: true },
+    noOfParticipants: { type: String, required: true },
+    date: { type: Date, required: true },
+    email: { type: String, required: true },
```

Comment:
- remove the email `email: { type: String, required: true },`  
- add reference to the EventOrganizer